### ISSUE TITLE
update plugin

### ISSUE THREAD

**Issue by andim27** [ShekarMudaliyar/social_share#159]
Please update plugin to the last Flutter(and gradle) rules.You need set new vars in build.gradle  and UP  gradle&kotlin version. Please make it as soon as possible !

**Comment by SubutaDan** [comment]
Hi @andim27 , Do you think the changes you have requested would resolve the problem I described in #158?

**Comment by andim27** [comment]
Yes it depends on last gradle (8.3 >=) version Make last flutter project Look into gradle setting and apply them to your plugin

**Comment by Nico04** [comment]
A related PR is opened : https://github.com/ShekarMudaliyar/social_share/pull/160
